android/src/main/java/com/onehilltech/selfsigned/android/AndroidSelfSigned.java

```diff
@@ -30,9 +30,20 @@ public static SSLContext newSSLContext (Context context, String assetFile)
   public static SSLContext newSSLContext (Context context, String type, String assetFile)
       throws CertificateException, IOException, KeyStoreException, NoSuchAlgorithmException, KeyManagementException
   {
-    CertificateFactory cf = CertificateFactory.getInstance (type);
     InputStream caInput = new BufferedInputStream (context.getAssets ().open (assetFile));
+    return newSSLContext(type, caInput);
```

Comment:
Please add a space before the open parenthesis.

Reply:
No problems, I've done it.